### PR TITLE
add summa bug

### PR DESCRIPTION
**Pull request by enricobottazzi**
Add new bug related to Summa

### CODE REVIEW

**Comment by kcharbo3** [comment]
Thanks for this! Excited to push this through.

**Comment by kcharbo3** [comment]
Before pushing through, let me know what you think about my last comment, thanks!

**Comment by enricobottazzi** [comment]
Hi @kcharbo3, I don't see your comment in the review

File: README.md [hunk 1]
```diff
@@ -876,6 +878,71 @@ require(_inputs[4]<zokratesPrime, "Input too large - possible overflow attack");
 1. [Github Issue](https://github.com/EYBlockchain/nightfall/issues/95)
 2. [Github Fix](https://github.com/EYBlockchain/nightfall/pull/96)
 
+
+## <a name="summa-1">19. Summa: Unconstrained Constants Assignemnt </a>
+
+**Summary**
+
+Related Vulnerabilities: 1. Under-constrained Circuits,
```
[the remark this answers is not in the document]
Comment (enricobottazzi) [reply README.md]: agree! added to latest commit

**Pull request merged**